sklearn/__init__.py

```diff
@@ -73,6 +73,12 @@
     # We are not importing the rest of scikit-learn during the build
     # process, as it may not be compiled yet
 else:
+    # Import numpy, scipy to make sure that the BLAS libs are loaded before
+    # creating the ThreadpoolController. (OpenMP is loaded by importing show_versions)
```

Comment:
Maybe it is useful to add the comment from https://github.com/scikit-learn/scikit-learn/pull/28838/files#r1567341343 that numpy and scipy would be imported later on anyway and that the point is to make sure that they are imported early

Reply:
Sorry I don't understand what you're asking. This comment is already saying that we import numpy and scipy early to make sure that the controller will find the BLAS libs, no ?

Reply:
When reading the code (for the first time) from top to bottom the "import early" didn't imply "we would anyway import this later". This is what made me think a comment saying that would be useful.

Reply:
I extended the comment to explain that this is to make the import explicit but they would be imported right after anyway